code/go/0chain.net/blobbercore/blobbergrpc/proto/blobber_contract.proto

```diff
@@ -370,6 +370,8 @@ message MarketplaceShareInfoRequest {
 }
 
 message MarketplaceShareInfoResponse {
-  string response_status = 1;
+  reserved 1;
```

Comment:
You can remove the reserved here as the following proto is not in prod, I dont think we may get any backward compatibility issue